src/Artsy/SystemContext.tsx

```diff
@@ -35,6 +35,11 @@ export interface SystemContextProps {
    * and `USER_ACCESS_TOKEN` environment variables if available.
    */
   user?: User
+
+  /**
+   * FIXME: Ask alloy how to pass one-off props like this in from force
+   */
+  notificationCount?: number
```

Comment:
@alloy - when you get back wondering if we can discuss approaches to passing data into the `SystemContext` from force. Now that we removed free-form access this will error out and I'm not sure how to extend it from the perspective of `<SystemContextProvider {...otherProps}>`.